packages/google_sign_in/lib/widgets.dart

```diff
@@ -38,6 +39,11 @@ class GoogleUserCircleAvatar extends StatelessWidget {
   /// The Google user's identity; guaranteed to be non-null.
   final GoogleIdentity identity;
 
+  /// The color of the text.
```

Comment:
This is only relevant if no photo to display is available, right? Might be worth documenting that.

Reply:
Done.